app/routes/suggest_translation.py

```
@@ -1,26 +1,43 @@
 from urllib import request
```

Ruff (0.8.2)

`urllib.request` imported but unused

Remove unused import: `urllib.request`

(F401)